app/Http/Controllers/FrontendTransportController.php

```diff
@@ -143,8 +143,10 @@ public function TrainCheckin(Request $request): RedirectResponse {
         $validated = $request->validate([
                                             'tripID'            => ['required'],
                                             'start'             => ['required', 'numeric'], //Origin station IBNR
+                                            //if the train has no departure, arrival is okay
                                             'departure'         => ['required', 'date'],
                                             'destination'       => ['required', 'numeric'], //Destination station IBNR
+                                            //if the train has no arrival, departure is okay
                                             'arrival'           => ['required', 'date'],
```

Comment:
I don't understand yet, how this works, and how the comments change anything in the 'required' formula, but LGTM.

Reply:
@MrKrisKrisu Please advice, after that, I'd merge it.

Reply:
I realize that these comments are more confusing than helpful. They only say that if no departure is available (e.g. because a train stops only to get off) the arrival is used as an alternative. I would remove the comments.